backend/apps/matching/templates/participant/participant_info_edit_form.html

```diff
@@ -14,23 +12,95 @@
+                    <br>
+                <hr>
+                    <h5>Your locations</h5>
+                    {{ location_formset.management_form }}
+                    <div id="form_set">
+                        {% for form in location_formset.forms %}
+                            {% if form.non_field_errors %}
+                            <div class="alert alert-danger" role="alert">
+                             {{form.non_field_errors }}
+                            </div>
+                            {% endif %}
+                            <div class="form-row">
+                            {% for hidden in form.hidden_fields %}
+                                {{ hidden }}
+                            {% endfor %}
+                            <div class="form-group col-md-2 mb-0">
+                            {{ form.country_code |as_crispy_field }}
+                            </div>
+                            <div class="form-group col-md-6 mb-0">
+                            {{ form.plz |as_crispy_field }}
+                            </div>
+                            <div class="form-group col-md-2 mb-0">
+                            {{ form.radius |as_crispy_field }}
+                            </div>
+                            <div class="form-group col-md-2 mb-0 align-middle">
+                                {{ form.DELETE |as_crispy_field }}
+                            </div>
+                            </div>
+                        {% endfor %}
+                    </div>
+                    <input type="button" class="btn btn-sm btn-outline-primary" value="Add another +" id="add_more">
+                    <input type="button" class="btn btn-sm btn-outline-dark" value="Remove last -" id="remove_this">
+                    <br>
+                    <div id="empty_form" style="display:none">
+                            <div class="form-row">
+                            <div class="form-group col-md-2 mb-0">
+                            {{ location_formset.empty_form.country_code |as_crispy_field }}
+                            </div>
+                            <div class="form-group col-md-6 mb-0">
+                            {{ location_formset.empty_form.plz |as_crispy_field }}
+                            </div>
+                            <div class="form-group col-md-2 mb-0">
+
+                            {{ location_formset.empty_form.radius |as_crispy_field }}
 
+                            </div>
+
+                            </div>
+                    </div>
+                <hr>
+                <input type="submit" class="btn btn-success float-right" value="Save">
+                </form>
             </div>
         </div>
 
     </div>
+
+    <!-- inspired from : https://simpleit.rocks/python/django/dynamic-add-form-with-add-button-in-django-modelformset-template/ -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Should probably also be included with webpack, @maltezacharias?

Reply:
If @maltezacharias  fixes this it would be good to merge then?

Reply:
@maltezacharias